scripts/player_logging.sh

```diff
@@ -9,7 +9,7 @@ get_steamid(){
 
 get_playername(){
     local player_info="${1}"
-    echo "${player_info}" | sed -E 's/,([0-9]+),[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]//g'
+    echo "${player_info}" | sed -E 's/,([0-9A-Z]+),[0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9]//g'
```

Comment:
Fixed an issue where player names could not be retrieved correctly.
It may be a fix for #538